PULL_REQUEST_TEMPLATE/breaking.md

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
maybe add link to spec issue/PR?

Reply:
it is implied in line 9:
`Keep it short... Any resolved issues?`

Reply:
it is implied in line 9:
`Keep it short... Any resolved issues?`